packages/core/scss/motion/index.scss

```diff
@@ -182,9 +182,7 @@ $kendo-transitions: map.merge($default-transitions, $kendo-transitions);
 
     @media (prefers-reduced-motion: reduce) {
         :root {
-            @each $key, $value in $kendo-durations {
-                --kendo-duration-#{$key}: 0.01ms;
-            }
+            --kendo-duration-global: 0.01ms;
```

Comment:
I might be mistaken, but my understanding is that users, who set _prefers-reduced-motion: reduce_ typically expect no motion at all, rather that just subtle or imperceptible motion. I imagine we've considered 0 instead of 0.01ms, but personally, I feel it would be more consistent with accessibility expectations to keep it at 0.